src/librustdoc/html/render/mod.rs

```diff
@@ -2594,6 +2595,21 @@ fn render_call_locations(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item) {
         id = id
     );
 
+    // Create a URL to a particular location in a reverse-dependency's source file
+    let link_to_loc = |call_data: &CallData, loc: &CallLocation| -> (String, String) {
+        let (line_lo, line_hi) = loc.call_expr.line_span;
+        let (anchor, title) = if line_lo == line_hi {
+            (format!("{}", line_lo + 1), format!("line {}", line_lo + 1))
```

Comment:
nit: format has a marginal amount of overhead here
```suggestion
            ((line_lo + 1).to_string(), format!("line {}", line_lo + 1))
```